DeckSwipe/ProjectSettings/UnityConnectSettings.asset

```diff
@@ -4,11 +4,12 @@
 UnityConnectSettings:
   m_ObjectHideFlags: 0
   serializedVersion: 1
-  m_Enabled: 0
+  m_Enabled: 1
```

Comment:
Does this enable Unity online services, where previously they'd be disabled? I feel like that doesn't need to be the default.

Reply:
Makes sense, but every time I build the project it changes back to `1`. Tried disabling the *CLOUD* option to *NOT CONNECTED* in Unity Hub for the project but it doesn't seem to have any impact on the `m_Enabled` property (turns back on after build).

Reply:
Fair enough